docs/configuring.md

```diff
@@ -365,6 +365,25 @@ copier --exclude '*' --exclude '!file-i-want' copy template destination
 
     Instead, CLI/API definitions **will extend** those from `copier.yml`.
 
+### `extensions`
```

Comment:
Do you think it makes sense to call this `jinja_extensions`? 🤔 

That maybe makes it more self-explanatory, and separates it from maybe copier extensions or other kind of extensions that might come in the future? Disclaimer: I'm not particularly thinking on doing that right now.

Reply:
We already have `envops` and not `jinja_envops`, but yeah, `jinja_extensions` is fine, it's more explicit :slightly_smiling_face: